qutebrowser/completion/models/urlmodel.py

```diff
@@ -86,6 +88,12 @@ def url(*, info):
         models['bookmarks'] = listcategory.ListCategory(
             'Bookmarks', bookmarks, delete_func=_delete_bookmark, sort=False)
 
+    if config.val.tabs.switch_to_tab_on_url_match and 'tabs' in categories:
```

Comment:
I wonder if this should just be `'tabs' in categories`. I'm not sure if there's a good use case for completing tabs if `switch_to_tab_on_url_match` isn't set, but it could be confusing if setting `tabs` alone doesn't control this category.

If we keep it like this, at least document that _both_ need to be set for the category to appear.

Reply:
Good questions. I don't think I can answer this myself :)